abstract_specification.md

```diff
@@ -649,6 +655,12 @@ Examples, using a pseudo-syntax:
 
 ## Platform-Specific Configuration
 
+SOMA includes provisions for two separate ways to provide platform-specific configuration data to objects and operations.
+For configuration settings that affect a single operation, SOMA functions include a `platform_config` parameter, which can be used on a call-by-call basis to specify short-term behavior.
+For long-lived configuration settings, SOMA objects expose a `context` field that contains implementation-defined data that can be shared over an entire analysis session.
```

Comment:
Perhaps: "For multi-operational configuration settings, ..."?

Reply:
Reworked this along with the below.

Reply:
It still begs the question of what an "analysis session" is, and how it relates to the lifespan of the object.  Can this be tightened to something like "... shared over the life of the object?"

Reply:
It can last beyond the lifecycle of any single object (cf. the manual-context-passing part of the example), so I'm not sure "the life of the object" quite captures it; I'll give this a little time to stew.